dataactbroker/handlers/fileHandler.py

```diff
@@ -509,6 +509,8 @@ def startGenerationJob(self, submission_id, file_type):
             jq.generate_d_file.delay(get_url, user_id, job.job_id, InterfaceHolder, timestamped_name, self.isLocal)
         else:
             # TODO add generate calls for E and F
+            job.job_status_id = jobDb.getJobStatusId("finished")
```

Comment:
instead of doing this directly in fileHandler, shouldn't this be calling a function in jobHandler, something similar to `setDFileStatus`?